src/app/components/LanguageInfo.svelte

```diff
@@ -47,6 +48,10 @@
     let { key, value } = event.detail;
     updateMetadata(key, value);
   }
+
+  function addCopyrightSymbol() {
+    copyright += "©";
```

Comment:
Should we add logic that only prepends the symbol once?

Reply:
Yes, and you are right that © goes before everything else